life_dashboard/dashboard/forms.py

```
@@ -1,15 +1,46 @@
+        fields = ("username", "email", "password1", "password2")
+
+    def save(self, commit=True):
+        user = super().save(commit=False)
+        user.email = self.cleaned_data["email"]
+        if commit:
+            user.save()
+        return user
+
+
+class UserProfileForm(forms.ModelForm):
+    class Meta:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)